src/ds_caselaw_utils/courts.py

```diff
@@ -59,6 +64,26 @@ def get_jurisdiction(self, code: str) -> Optional[Jurisdiction]:
     def expand_jurisdictions(self) -> list["Court"]:
         return [self] + [CourtWithJurisdiction(self, jurisdiction) for jurisdiction in self.jurisdictions]
 
+    def _render_markdown_text(self, type: str) -> Optional[str]:
+        if not self.canonical_param:
+            return None
+
+        filename = self.canonical_param.replace("/", "_")
+        description_md_file_path = pathlib.Path(__file__).parent / f"data/markdown/{type}/{filename}.md"
+        try:
+            with open(description_md_file_path) as file:
+                return str(md.render(file.read()))
+        except FileNotFoundError:
+            return None
+
+    @cached_property
+    def description_text_as_html(self) -> Optional[str]:
+        return self._render_markdown_text("description")
+
+    @cached_property
+    def old_documents_support_text_as_html(self) -> Optional[str]:
```

Comment:
Would "historic" be a better word? "old" might just be "we didn't quite finish refactoring this code"